qiskit_nature/problems/second_quantization/molecular/molecular_problem.py

```diff
@@ -0,0 +1,92 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The Molecular Problem class."""
+from typing import List, Optional, Tuple
+
+from qiskit_nature.drivers.qmolecule import QMolecule
+from qiskit_nature.drivers import FermionicDriver
+from qiskit_nature.operators import FermionicOp
+from qiskit_nature.operators.second_quantization import SecondQuantizedOp
+from qiskit_nature.transformers import BaseTransformer
+from .integrals_calculators.angular_momentum_integrals_calculator import \
+    calc_total_ang_momentum_ints
+from .fermionic_op_builder import build_fermionic_op, build_ferm_op_from_ints
+from .integrals_calculators.magnetization_integrals_calculator import calc_total_magnetization_ints
+from .integrals_calculators.particle_number_integrals_calculator import calc_total_particle_num_ints
```

Comment:
Goes back to my comment above about init files this could be from .integrals_calculators import...